### PR TITLE
Update presence.ts

### PR DESCRIPTION
**Pull request by Isqneeh**
I missed this page in my rich presence

### CODE REVIEW

File: websites/H/HackerRank/presence.ts [hunk 1]
```diff
@@ -32,6 +32,9 @@ presence.on("UpdateData", () => {
     presenceData.details = "Viewing inbox";
   } else if (path("/notifications")) {
     presenceData.details = "Viewing notifications";
+  } else if (path("/contests")) {
+    presenceData.details = "Viewing contests";
+  }
   } else if (
```
Comment (ririxidev) [inline websites/H/HackerRank/presence.ts:38]: um what?

**Review by ririxidev — CHANGES_REQUESTED** [review]
and bump version and upload proof of owrking presence on it

**Review by Bas950 — CHANGES_REQUESTED** [review]
Bump version upload proof

**Comment by ririxidev** [comment]
No response from pull request creator. Closing.

**Pull request closed without merging**